openscm/adapters/__init__.py

```diff
@@ -52,5 +54,6 @@ def load_adapter(name: str) -> type:
         if adapter is None:
             raise KeyError("Unknown model '{}'".format(name))
         else:  # pragma: no cover
```

Comment:
this should already be hit by `tests/unit/test_adapter_registry` no?

Reply:
No, we don't have a working model adapter yet (I added the TODO comment below)